package.json

```diff
@@ -1,38 +1,30 @@
 {
-  "name": "react-certification-2020",
```

Comment:
Could I ask why did you removed the starting point of the app? the installed packages were helping you for the formatting. It is not a bad option removing all the content, is just curiosity.